### ISSUE TITLE
Make language toggle more accessible for screen readers

### ISSUE THREAD

**Issue by alexandrevryghem** [DSpace/dspace-angular#4267]
## Describe the bug
The language switch button should clearly state the currently selected language option using the `aria-selected` attribute. We should also consider switching the language toggle to a `<button>` instead of an `<a>` to be more semantically correct

## Expected behavior
Screen readers should be able to detect which option is currently selected
